.github/workflows/deployment.yaml

```
@@ -0,0 +1,22 @@
+name: Deployment
+
+on:
+  workflow_dispatch:
+
+jobs:
+  deployment:
+    name: Deployment
+    runs-on: ubuntu-latest
+    environment: production
+
+    steps:
+    - name: Checkout repo
+      uses: actions/checkout@v4
+    - name: Fly actions
+      uses: superfly/flyctl-actions/setup-flyctl@master
+      with:
+        version: 0.0.462
+    - name: Fly deploy
+      run: flyctl deploy --remote-only
+      env:
+        FLY_API_TOKEN: ${{ secrets.FLY_API_TOKEN }}
```

Please add a deeper indentation to make the steps clearly visible.
```suggestion
    steps:
      - name: Checkout repo
        uses: actions/checkout@v4
      - name: Fly actions
        uses: superfly/flyctl-actions/setup-flyctl@master
        with:
          version: 0.0.462
      - name: Fly deploy
        run: flyctl deploy --remote-only
        env:
          FLY_API_TOKEN: ${{ secrets.FLY_API_TOKEN }}
```